routes/conversation.js

```diff
@@ -169,6 +169,8 @@ exports.leave = function (req, res){
 			var participantIds = convo.participants.map(function (p){return p._id});
 			var userString = JSON.stringify(req.user._id);
 			var participantIdStrings = participantIds.map(function (id){return JSON.stringify(id)});
+			// Should this 'found' variable definition be changed to the equivalent 'foundIndex' variable
+			// definitions used below in 'exports.archive'?
```

Comment:
@psybuzz I was looking through the leave functionality, and it seems like we use `found` as a boolean here and then delete one entry starting at index `found` to remove the target user from the conversation. If `found` is cast as a boolean, wouldn't the `convo.participants.splice(found, 1);` line inside `if (found){...}` always remove the first entry in the list? 

I wanted to bring this up because in the archive functionality below I'm saving `foundIndex` as an integer and then checking that this integer is valid (as opposed to storing a boolean). Should we adopt a similar approach here as well?